packages/vuetify/src/components/VCombobox/VCombobox.ts

```diff
@@ -108,11 +108,6 @@ export default VAutocomplete.extend({
 
       this.$nextTick(this.updateSelf)
     },
-    onFilteredItemsChanged (val: never[], oldVal: never[]) {
-      if (!this.autoSelectFirst) return
```

Comment:
The only purpose of this line code seems to be allowing entering a custom input as a final result, for example, if a list has element of "bar" but user inputs "ba" and hit enter, it should add "ba" as the final result. 

The tricky part is when user inputs "ba" it automatically selects the first option "bar" (due to line 231 in `VAutocomplete`) and when user hits enter it will set "bar" as final result.

However, this line of code is unnecessary, a proper fix is it should only select the first item when `autoSelectFirst=true` in `VAutocomplete`. This will make sure there is no default selection in `combobox` when `autoSelectFirst=false`